### PR TITLE
Bep-174: Implement Cross Chain Relayer Management

### PR DESCRIPTION
**Pull request by emailtovamos**
### Description

This PR incorporates required changes for [BEP174](https://github.com/bnb-chain/BEPs/pull/174) which is mostly about updating of the bytecode of RelayerHub system smart contract. This will be part of the Bohr Hardfork.

### Rationale

The BEP will improve the management of the relayer whitelist by introducing Relayer Managers, where a Relayer Manager can manage the registration of a single Relayer.
Managers will be elected and ejected via governance.

### Changes

Notable changes: 
* Upgrading of bytecode of RelayerHub smart contract across networks.

### CODE REVIEW

File: core/vm/contracts.go [hunk 1]
```diff
@@ -168,11 +184,17 @@ func init() {
 	for k := range PrecompiledContractsIsMoran {
 		PrecompiledAddressesMoran = append(PrecompiledAddressesMoran, k)
 	}
+
+	for k := range PrecompiledContractsIsBEP174 {
```
Comment (guagualvcha) [inline core/vm/contracts.go:188]: I think this is not necessary change.

File: core/vm/contracts.go [hunk 2]
```diff
@@ -140,6 +155,7 @@ var PrecompiledContractsBLS = map[common.Address]PrecompiledContract{
 }
 
 var (
+	PrecompiledAddressesBEP174    []common.Address
```
Comment (guagualvcha) [inline core/vm/contracts.go:158]: No need to define PrecompiledAddressesBEP174.

**Review by guagualvcha — CHANGES_REQUESTED** [review]
The PR title, description and commit info does not follow the standard, please make some change.  Example PR: https://github.com/bnb-chain/bsc/pull/1301/commits

**Comment by guagualvcha** [comment]
I change the base branch to develop, so please take care of the commits in your branch.

**Comment by guagualvcha** [comment]
Fix the CI check as well. Thanks.

**Review by guagualvcha — APPROVED** [review]

**Review by brilliant-lx — APPROVED** [review]

**Pull request merged**